api/client/service/update.go

```diff
@@ -370,6 +380,29 @@ func updateMounts(flags *pflag.FlagSet, mounts *[]mounttypes.Mount) {
 	*mounts = newMounts
 }
 
+func updateGroups(flags *pflag.FlagSet, groups *[]string) error {
```

Comment:
@cpuguy83 I think golang is pass by value. The passed `groups` is updated so used a pointer to change the slice.